src/main/java/com/spring/app/domain/Company.java

```diff
@@ -6,26 +6,30 @@
 import javax.persistence.*;
 import java.util.Set;
 
+/**
+ * Company
+ * Has a one-directional relation bound to country
+ * @author lyubov
+ */
 @Data
 @Entity
 @Builder
 @NoArgsConstructor
```

Comment:
ho-ho-ho, Do you know that @ Data includes everything ?)

Are you sure that your entity needs all generated code?